projects/manuals-publisher/docker-compose.yml

```diff
@@ -29,3 +29,34 @@ services:
       TEST_MONGODB_URI: "mongodb://mongo-3.6/manuals-publisher-test"
       REDIS_URL: redis://redis
 
+  manuals-publisher-app: &manuals-publisher-app
+    <<: *manuals-publisher
+    depends_on:
+      - mongo-3.6
+      - redis
```

Comment:
Could you reorder `redis` and `mongo-3.6` on lines 25-26, so that they're consistent throughout the file? It makes it slightly more obvious what's different between the images.

Reply:
Sure thing 👍